### PR TITLE
Update download urls

### LINKED ISSUE #1899: Maps not auto downloading
**Describe the bug**
Maps do not auto download when joining a game due to the server change returning 404 errors for the current urls

**To Reproduce**
join a game with a map you dont have

**Expected behavior**
Should download maps before joining lobby. Vault works fine so just need to change the url

### PR DESCRIPTION
**Pull request by Sheikah45**
Fixes #1899 
fixes download urls

### CODE REVIEW

**Comment by Brutus5000** [comment]
Previews, mods and replays also affected, basically everything that contains the path /faf/vault

**Comment by Brutus5000** [comment]
I rechecked it an modDownloadUrlFormat is obsolete and can be removed from the yaml and the properties. I noticed that because the yaml points to the map directory instead.

**Comment by Sheikah45** [comment]
Is it reasonable for links to be fixed by just removing /faf/vault from the url?
Also the modDownloadUrlFormat might be named for clarity in what it is used for but am not sure if it is needed.

File: src/main/resources/application-prod.yml [hunk 1]
```diff
@@ -26,6 +26,6 @@ faf-client:
   status-page-url: https://statuspage.faforever.com
 
   forged-alliance:
-    exe-url: https://content.faforever.com/faf/updaterNew/updates_faf_files/ForgedAlliance.exe
+    exe-url: https://content.faforever.com/updaterNew/updates_faf_files/ForgedAlliance.exe
```
Comment (1-alex98) [inline src/main/resources/application-prod.yml]: like that @Brutus5000 ?

Comment (1-alex98) [reply src/main/resources/application-prod.yml]: does not work so i guess not

Comment (Sheikah45) [reply src/main/resources/application-prod.yml]: think it was only /faf/vault

**Review by 1-alex98 — DISMISSED** [review]
works

File: src/main/resources/application-prod.yml (hunk 1 above)
Comment (Sheikah45) [reply src/main/resources/application-prod.yml]: just /faf is fine

Comment (Brutus5000) [reply src/main/resources/application-prod.yml]: Indeed /faf is fine. It was just that the /faf/vault/\<folder> was actually a symlink on the server to ../../\<folder> so I removed it

**Review by Brutus5000 — CHANGES_REQUESTED** [review]
modDownloadUrlFormat can removed completely as I said. (Also having it point to maps is super confusing)

**Comment by Sheikah45** [comment]
So then should the mod vault just call getMapDownloadURL or is it better to rename the link?

**Comment by Brutus5000** [comment]
According to my IntelliJ analysis the variable is not used anywhere. What do you want to rename where?

**Comment by Sheikah45** [comment]
Hmm I thought it was used by the mod but looks like I was wrong

**Review by Brutus5000 — APPROVED** [review]

**Pull request merged**